### PR TITLE
Fuji rx2540

### PR DESCRIPTION
**Pull request by hooligan-sa**
Adds Fujitsu PRIMERGY RX2540 M5 device

### CODE REVIEW

**Comment by hooligan-sa** [comment]
I don't seem to have the ability to remove the 'stale' label

File: device-types/Fujitsu/primergy-rx2540-m5.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+airflow: front-to-rear
+is_full_depth: true
+weight: 25
+weight_unit: kg
+comments: '[Fujitsu Server PRIMERGY RX2540 M5 Rack Server](https://www.fujitsu.com/vn/en/imagesgig5/Data_Sheet_FUJITSU_Server_PRIMERGY_RX2540_M5_Rack_Server.pdf)'
+console-ports:
+  - name: Serial
+    type: de-9
+  - name: VGA
+    type: other
```
Comment (danner26) [inline device-types/Fujitsu/primergy-rx2540-m5.yml:15]: VGA is not a console port, please remove this definition.

File: device-types/Fujitsu/primergy-rx2540-m5.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+power-ports:
+  - name: Power 1
+    type: iec-60320-c14
+    maximum_draw: 715
+  - name: Power 2
+    type: iec-60320-c14
+    maximum_draw: 715
```
Comment (danner26) [inline device-types/Fujitsu/primergy-rx2540-m5.yml:22]: It looks like these PSUs might be modular. If this is the case, the proper way to model this is to add them under `module-bays` and then create the associated modular PSUs under the module-types folder

**Review by danner26 — CHANGES_REQUESTED** [review]

File: device-types/Fujitsu/primergy-rx2540-m5.yml (hunk 2 above)
Comment (hooligan-sa) [reply device-types/Fujitsu/primergy-rx2540-m5.yml]: Converted to module-bays

File: device-types/Fujitsu/primergy-rx2540-m5.yml (hunk 1 above)
Comment (hooligan-sa) [reply device-types/Fujitsu/primergy-rx2540-m5.yml]: I just used the same info from all the other modules in the Fujitsu directory, but removed - Thank you

**Review by hooligan-sa — COMMENTED** [review]
Changes completed, please re-evaluate @danner26

**Review by danner26 — APPROVED** [review]

**Pull request merged**